src/vm/wellknownattributes.h

```diff
@@ -0,0 +1,107 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#ifndef __WELLKNOWNATTRIBUTES_H_
+#define __WELLKNOWNATTRIBUTES_H_
+
+enum class WellKnownAttribute : DWORD
```

Comment:
This is only attributes that the VM/JIT cares about, correct?

Reply:
At the moment. Yeah, its most of the ones the VM/JIT cares about. @sebastienros suggested I look into ASP.NET startup and see if I should see if I thought this might be useful to improve attribute query perf for managed code too.